site/src/server.rs

```diff
@@ -594,7 +594,11 @@ lazy_static::lazy_static! {
 }
 
 /// Handle the case where the path is to a static file
-async fn handle_fs_path(req: &Request, path: &str) -> Option<http::Response<hyper::Body>> {
+async fn handle_fs_path(
+    req: &Request,
+    path: &str,
+    allow_compression: bool,
```

Comment:
```suggestion
    use_compression: bool,
```
I think that this more clearly states what we want to do. Same in `get_static_asset`.